### ISSUE TITLE
[BUG] `placeTypesFilter` parameter not working in `findAutocompletePredictions`

### ISSUE THREAD

**Issue by esuljic** [matanshukry/flutter_google_places_sdk#57]
I make the following call, but the results I am getting are not only the `REGIONS`. I am getting all the businesses as cafes as well.
```
_places.findAutocompletePredictions(
        searchQuery.text,
        countries: ['ch'],
        placeTypesFilter: [PlaceTypeFilter.REGIONS],
      )
```

Tested on iOS simulator. I tried putting `GEOCODE` and `CITIES`, but I got all the results regardless.

I saw that [this enum](https://developers.google.com/maps/documentation/places/android-sdk/reference/com/google/android/libraries/places/api/model/TypeFilter) is deprecated, but there is no alternative option in this Flutter package.

**Comment by esuljic** [comment]
It is a duplicate of #48.

**Comment by matanshukry** [comment]
Fixed in 0.3.7 ( see https://github.com/matanshukry/flutter_google_places_sdk/pull/59 for more info )
